config/server.properties

```diff
@@ -129,3 +129,10 @@ zookeeper.connect=localhost:2181
 zookeeper.connection.timeout.ms=6000
 
 
+############################# Group Coordinator Settings #############################
+
+# The following configuration specifies the time, in milliseconds, that the GroupCoordinator will delay the initial consumer rebalance.
+# The rebalance will be further delayed by the value of group.initial.rebalance.delay.ms as new members join the group, up to a maximum of <code>max.poll.interval.ms</code>.
+# The default value for this is 3 seconds.
+# During development and testing it might be desirable to set this to 0 inorder to not delay test execution time.
```

Comment:
I think we should edit the last line and make it relevant to this case. That is, explain why we set it to `0` here and that the default of `3` seconds is more suitable for production (i.e. this line should be removed)